packages/FormElement/stories/examples/ButtonGroup.js

```diff
@@ -1,27 +1,38 @@
 import React from "react";
 import ButtonGroup from "@paprika/button-group";
-import stylers from "@paprika/stylers";
+import L10n from "@paprika/l10n";
 import FormElement from "../../src";
 
-const { Content, Label } = FormElement;
+const { Content, Label, Description } = FormElement;
 
 export default function ButtonGroupExample() {
   const refButtonGroup = React.useRef(null);
-  const buttonOptionsArray = ["Black Panther", "Wonder Woman", "Spiderman"];
-  const buttonGroupOptions = buttonOptionsArray.map(hero => (
-    <ButtonGroup.Item value={hero} key={hero}>
-      {hero}
-    </ButtonGroup.Item>
-  ));
+  const labelId = "label-buttongroup";
+  const buttonOptions = ["Batman", "Wolverine", "Daredevil"];
```

Comment:
What do you have against Black Panther, Wonder woman and Spiderman. They are all very capable super heroes.

Reply:
Variety is the Paprika of life.